contributing.md

```diff
@@ -76,6 +76,117 @@ pre-commit run --all-files
 The `VERSION` file at the root of the repo is the single source for `legate-boost`'s version.
 Modify that file to change the version for wheels, conda packages, the CMake project, etc.
 
+## Work with the conda packages
```

Comment:
These kind of detailed build instructions go stale very fast. Just an observation, not a complaint :)

Reply:
Agreed, I hope we can simplify it in the future. Some of this can be simplified with tooling... including using more things from RAPIDS.

But I wanted to try to keep things on GitHub's artifact store for this PR, as using the RAPIDS tools involves some other architectural changes that I wasn't sure how to fit in cleanly here.